pulumi-eks-go/main.go

```diff
@@ -0,0 +1,439 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/pulumi/pulumi-aws/sdk/v4/go/aws/ec2"
+	"github.com/pulumi/pulumi-aws/sdk/v4/go/aws/eks"
+	"github.com/pulumi/pulumi-aws/sdk/v4/go/aws/iam"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		prefix := "pulumi-eks-go"
+
+		// Resource: VPC
+		// Purpose: Amazon Virtual Private Cloud (Amazon VPC) enables you to launch AWS resources into a virtual network that you've defined.
+		// Docs: https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html
+
+		// VPC CIDR
+		cidrBlock := "10.0.0.0/16"
+
+		// VPC Args
+		vpcArgs := &ec2.VpcArgs{
+			CidrBlock:          pulumi.String(cidrBlock),
+			EnableDnsHostnames: pulumi.Bool(true),
+			InstanceTenancy:    pulumi.String("default"),
+			Tags: pulumi.StringMap{
+				"Name": pulumi.String(prefix + "-vpc"),
```

Comment:
Need to find out if I can define a map ahead of time and change the `Name` key each time - would like to have 4/5 additional tags but feels redundant adding them each time

Reply:
Resolved: https://github.com/gsweene2/pulumi/pull/1/commits/716b247c4698914bd82ba60f3b947f3488af4bed